hivemq-edge/src/frontend/src/extensions/datahub/components/controls/DeleteListener.spec.cy.tsx

```diff
@@ -132,4 +134,25 @@ describe('DeleteListener', () => {
       cy.wrap(w[3]).should('contain.text', 0)
     })
   })
+
+  it.only('should not delete if the designer is readonly', () => {
```

Comment:
suggestion(blocking): watch out there is an `only` here

```suggestion
  it('should not delete if the designer is readonly', () => {
```